gemfiles/mime_types_2.latest.gemfile

```diff
@@ -1,12 +0,0 @@
-# This file was generated by Appraisal
-
-source "https://rubygems.org"
-
-gem "jruby-openssl", :platforms => :jruby
-gem "mime-types", "~> 2.3"
-
-group :local_development, :test do
-  gem "appraisal", "~> 1.0"
-end
-
-gemspec :path => "../"
```

Comment:
Why'd you remove this gemfile? It guarantees that we're testing the latest 2.x mime-types.